src/NuGet.Core/NuGet.CommandLine.XPlat/Strings.resx

```diff
@@ -220,4 +220,7 @@
   <data name="Restore_Switch_NoCache_Description" xml:space="preserve">
     <value>Do not cache packages and http requests.</value>
   </data>
+  <data name="Restore_Switch_IgnoreFailedSource_Description" xml:space="preserve">
+    <value>Ignore Failed Source during restore.</value>
```

Comment:
fix the capitalization - only capitalize the first word
Explain what ignore means (warn? quiet?)